docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -343,12 +403,39 @@ Below options are used to configure Azure Service Bus SDK Client.
 
 |*spring.cloud.azure.servicebus*.connection-string | String | Service Bus Namespace connection string value.
 |*spring.cloud.azure.servicebus*.namespace | String | Service Bus Namespace value.
-|*spring.cloud.azure.servicebus*.domainName | String | Domain name of an Azure Service Bus Namespace value.
-|*spring.cloud.azure.servicebus*.entityName | String | Entity name of Azure Service Bus queue or topic.
-|*spring.cloud.azure.servicebus*.entityType | ServiceBusEntityType | Entity type of Azure Service Bus queue or topic.
-|*spring.cloud.azure.servicebus*.crossEntityTransactions | Boolean | Enable cross entity transaction on the connection to Service bus.
+|*spring.cloud.azure.servicebus*.domain-name | String | Domain name of an Azure Service Bus
+Namespace value.
+|*spring.cloud.azure.servicebus*.entity-name | String | Entity name of Azure Service Bus queue or
+topic.
+|*spring.cloud.azure.servicebus*.entity-type | ServiceBusEntityType | Entity type of Azure
+Service Bus queue or topic.
+|*spring.cloud.azure.servicebus*.cross-entity-transactions | Boolean | Enable cross entity
+transaction on the connection to Service bus.
+|*spring.cloud.azure.servicebus*.producer.* | see Notes | Producer configuration options.
+|*spring.cloud.azure.servicebus*.consumer.* | see Notes | Consumer configuration options.
+|*spring.cloud.azure.servicebus*.consumer.sessionEnabled | Boolean | Consumer Whether session is
+enabled.
+|*spring.cloud.azure.servicebus*.consumer.autoComplete | Boolean | Consumer whether auto complete
+flag.
+|*spring.cloud.azure.servicebus*.consumer.prefetchCount | Integer | Consumer prefetch count.
+|*spring.cloud.azure.servicebus*.consumer.subQueue | String | Consumer sub queue name.
+|*spring.cloud.azure.servicebus*.consumer.subscriptionName | String | Consumer subscription name.
+|*spring.cloud.azure.servicebus*.consumer.maxAutoLockRenewDuration | Duration | Consumer max
+duration for auto lock renew.
+|*spring.cloud.azure.servicebus*.consumer.receiveMode | String | Consumer receive mode.
+|*spring.cloud.azure.servicebus*.processor.*| see Notes | Processor configuration option.
```

Comment:
the vars of com.azure.spring.cloud.autoconfigure.servicebus.properties.AzureServiceBusProperties.Processor and com.azure.spring.cloud.autoconfigure.servicebus.properties.AzureServiceBusProperties.Consumer should be listed under *spring.cloud.azure.servicebus*.processor.xxx